pythonzm_Ops_task_views_41.py

```
@@ -0,0 +1,412 @@
+def path_del(request):
+    if request.method == 'POST':
+        name = request.POST.get('name')
+        p_name = request.POST.get('p_name')
+        path = os.path.join(p_name, name)
+        try:
+            if os.path.exists(path):
+                if os.path.isdir(path):
+                    shutil.rmtree(path)
+                else:
+                    os.remove(path)
+            return JsonResponse({'code': 200, 'msg': '删除成功！'})
```

Ruff (0.13.1)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)